pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -281,9 +306,12 @@ def newpkgcb(pkgId, name, arch):
                             await self.put(dc)
 
 
-class ErratumContentSaver(ContentSaver):
+class RpmContentSaver(ContentSaver):
```

Comment:
I renamed this one to make it look like a custom ContentSaver stage for RPM plugin. With the old name it sounds like this stage takes care of Errata content only.